app/controllers/ConfirmTransferController.scala

```diff
@@ -6,6 +6,8 @@ import java.util.UUID
 import configuration.{GraphQLConfiguration, KeycloakConfiguration}
 import graphql.codegen.AddFinalTransferConfirmation.AddFinalTransferConfirmation
 import graphql.codegen.types.AddFinalTransferConfirmationInput
+import graphql.codegen.AddFinalJudgmentTransferConfirmation.AddFinalJudgmentTransferConfirmation
```

Comment:
In order to make it cleaner/shorter, could you import `AddFinalJudgmentTransferConfirmation` like this

`graphql.codegen.AddFinalJudgmentTransferConfirmation.{AddFinalJudgmentTransferConfirmation => afjtc}`